app/public/views/userView/data/pageSettings/pageSettings.js

```diff
@@ -60,6 +61,36 @@ export default function pageSettings(model, close) {
         }
     }
 
+    function handleSidebarPreferenceSelection() {
+        const sidebar = document.getElementById('sidebar');
+        const visibleSidebarClassName = 'sidebar-visible';
+        const collapsibleSidebarClassName = 'sidebar-collapsible';
+        const collapsible = document.getElementsByClassName(collapsibleSidebarClassName);
+        const visible = document.getElementsByClassName(visibleSidebarClassName);
+        const sidebarPreferenceSelection = document.getElementById(sidebarPreferenceSelectId);
+        const selectedPreference = sidebarPreferenceSelection.options[sidebarPreferenceSelection.selectedIndex].value;
+        switch (selectedPreference) {
+            case '0':
+                /* Collapsible */
+                if (collapsible.length) {
+                    return;
+                }
+                sidebar?.classList.remove(visibleSidebarClassName);
+                sidebar?.classList.add(collapsibleSidebarClassName);
+                break;
+            case '1':
+                /* Always visible */
+                if (visible.length) {
+                    return;
+                }
+                sidebar?.classList.remove(collapsibleSidebarClassName);
+                sidebar?.classList.add(visibleSidebarClassName);
+                break;
```

Comment:
using more descriptive values, other than '0' and '1' would be better

Reply:
Done